docs/source/index.rst

```diff
@@ -182,6 +182,14 @@ Or directly from conda
 
    regression/*
 
+.. toctree::
```

Comment:
Could we make this order alphabetical?
I know it is not right now, so maybe this is fine and then we update in future PR